static/html/head.html

```diff
@@ -10,6 +10,6 @@
   <link
       href="https://fonts.googleapis.com/css2?family=Big+Shoulders+Inline+Display:wght@100..900&display=swap|family=Big+Shoulders+Text:wght@100..900&display=swap|family=Lora:ital,wght@0,400;0,700;1,400;1,700&display=swap"
       rel="stylesheet"/>
-  <link rel="stylesheet" href="/static/css/style.css"/>
+  <link rel="stylesheet" href="{{ .Stylesheet }}"/>
```

Comment:
This is a placeholder inside of a placeholder now. `Stylesheet` inside of `Head` inside of `Index`.

But should make swapping out settings way easier to have them in one place.